test/Customer-test.js

```diff
@@ -32,4 +34,23 @@ describe('Customer', () => {
     it('should have a name that is a string', () => {
       expect(customer.name).to.be.a('string');
```

Comment:
Also test for the value, not just the type.  Hate to get a name back like '58493'